### ISSUE TITLE
Propagation of Uncertainty

### ISSUE THREAD

**Issue by dwsideriusNIST** [dmzuckerman/Sampling-Uncertainty#42]
Per reviewers:

"In section 7.4 (propagation of uncertainty) expressions for the uncertainty of derived quantities are derived. While it is nice to see the derivation, I personally don’t think it is very necessary as the aim of the manuscript is to give basic orientation in the field of uncertainty analysis. It is the only derivation in the whole manuscript."

"However, section 7.4 is completely omitting the important topic of error propagation through multiple steps of simulations/experiments. I think it is very important to properly quantify errors if multiple simulation steps are needed calculate a certain quantity (this happens often in the field of free energy calculations). While the propagation of the experimental standard deviation of the mean might be straightforward, the authors should explain how such error propagation works if the uncertainty is quantified with confidence intervals, as the authors suggest."

**Comment by mangiapasta** [comment]
Okay let's discuss this a bit.  If you know how to propagate error through one step, don't you know how to do it through multiple steps...?

Also, use of confidence intervals doesn't really change anything does it?  If you're doing linear propagation, then your \delta x about some mean x is just going to represent the endpoints of the confidence interval, no?

If, on the other hand, you have to propagate error in one simulation into another, you're either going to use a Monte Carlo based sampling method or a more refined quadrature method such as polynomial chaos or a related Gaussian quadrature.  As far as the latter goes, my understanding is that it's still something of an active area of research as to how to use quadrature methods to propagate uncertainty between simulation realms.  At least I know of a few open problems in this realm.  Am I missing something though?

**Comment by mangiapasta** [comment]
Okay I may put a few sentences in on polynomial chaos / surrogate modeling, but only enough to point to relevant issues and the fact that these are open questions.

I like the derivation as is

**Comment by mangiapasta** [comment]
Okay, I addressed this issue in a new subsection at the end of Section 7.  If it's inappropriate, just take it out.  I tried to keep the discussion short and sweet, basically just pointing out obvious extensions of linear analyses and highlighting that harder propagation problems are active research.  I pointed to Ralph Smith's book on UQ and a few other papers where force-field propagation was incorporated into final uncertainty estimates.
